### ISSUE TITLE
Canada old age security benefits tax rules

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-canada#13]

**Comment by alex-rand** [comment]
@MaxGhenis this benefit depends on how long a person has lived in Canada (IE 'time since immigration'). The SPSD has this as a variable (see page 162 of the SPSD/M 'variable guide.pdf'), but I can't remember if we make the stronger assumption that everyone is just a born Canadian who has been there forever?

**Comment by MaxGhenis** [comment]
Let's make a `years_in_canada` variable, or if SPSD/M has a similarly styled variable name we can use that. Thanks Alex.

**Comment by alex-rand** [comment]
The amount calculation depends on this concept of 'net income', which is just total income minus a bunch of deductions. I've created a new issue for those deductions. 

In the meantime, what's the best way to write a placeholder formula for net income? I've tried:

`formula = total_individual_pre_tax_income - deductions_from_total_to_net_income`

Where `deductions_from_total_to_net_income` is itself a variable with:

`formula = 0`

Does this make sense as a temporary approach?
